### ISSUE TITLE
Can't see the whole window on a 1366x768 laptop screen...

### ISSUE THREAD

**Issue by gumbo-cms** [MaartenBaert/ssr#569]
Hi, well this program is great and I just got to record a great screencast in no time, only, I've had to fetch a second monitor to be able to do so... I justify:

I have a laptop, hp, 1366x788, which I use as my primary device. Once I've installed simplescreenrecorder and launched it, I wasn't able to see the entire window, even right-clicking on it and selecting move...
So, I've plugged in an external monitor I have, a 1920x1080 Samsung, and then I was able to see the whole window and record my screencast...

Sounds stupid, that's really exactly how it happened. So is there a way I get to use simplescreenrecoder on my primary 1366x768 laptop? (I'm using version 3.8-4 on fedora 26 x86_64)

Thanks

**Comment by QuImUfu** [comment]
duplicate of #374
possible fixes (i know):
 - install newest master branch
 - drag the window around using some key combo. (window manager specific)
 - virtually increase screen size (ssr will record in higher resolution as well, which is pretty awesome, i can record a 4k minecraft video while i am only playing on a 1280x1024 monitor)

**Comment by gumbo-cms** [comment]
okay thanks, actually alt+f7 works fine for me. alt+f8 also works but is trickier to use though

**Comment by gumbo-cms** [comment]
also, passed this issue, are there any incentives to use the git master version as opposed to fedora repo version?

**Comment by QuImUfu** [comment]
There are some other things, just look at the commits since Nov 7, 2016 (last version release), but nothing big.

**Comment by gumbo-cms** [comment]
Fair enough, now I've tried installing it yesterday and went through a lot of issues, ffmpeg-related. I'm just gonna wait for the new repo version, hoping it's going to come soon.
